README.md

```diff
@@ -1,19 +1,53 @@
 # Analysis of the State of the AnVIL 2024 Poll
 Analysis of the State of the AnVIL 2024 Poll
 
-## Demographics
+## Purpose
+This repository contains the analyses for the State of the AnVIL 2024 Poll and using the [OTTR website template](https://github.com/jhudsl/OTTR_Template_Website) renders a [website that provides an overview of the main results](https://hutchdatascience.org/AnVIL_Poll_2024/) for each section of the poll.
 
-### Institutional affiliation
+## Repository file structure
 
-![Institutional affiliation](https://github.com/fhdsl/anvilPoll2024/blob/3c6a05514355bce61033481470f940645928b83e/plots/institutionalType_faceteduserType.png)
+### root directory `.Rmd` files
 
-### Highest Degree attained
-![Highest Degree attained](plots/degree_usertype.png)
+* `TidyData.Rmd` -- this `.Rmd` file reads in the raw data using the `googlesheets4` API and then goes through tidying steps such as deduplicating responses based on email, setting simplified column names, recoding responses, etc. Every other `.Rmd` file in the root directory and the `pages` subdirectory depend upon this file and its `resultsTidy` dataframe.
+* `anvilPoll2024MainAnalysis.Rmd` -- this `.Rmd` file inherits the `resultsTidy` dataframe from `TidyData.Rmd` and performs the main analyses, saving the preferred plots, for each of the major questions posed in the poll.
+* `anvilPoll2024ExtraAnalysis.Rmd` -- This `.Rmd` file inherits the `resultsTidy` dataframe and main analyses from `anvilPoll2024MainAnalysis.Rmd` and runs supplemental analyses and plots extra plots on top of the main analysis.
 
-### Genomics and Clinical Research Experience
-![Research Experience](https://github.com/fhdsl/anvilPoll2024/blob/9178a9e2ca527eab98b8caeb3b31346e916113ab/plots/researchExperienceLevel_colorExperienceLevel_noUserTypeSplit.png)
+### other root directory files of importance
 
-## Insights
+* `config_automation.yml` -- this file specifies which docker image to use for this repo (we made a specific one instead of using default) and further turns on/off various OTTR PR check or rendering steps
```

Comment:
```suggestion
* `config_automation.yml` -- this file specifies which docker image to use for this repo (we made a [specific one](https://hub.docker.com/repository/docker/jhudsl/anvil-poll-2024/general) instead of using default) and further turns on/off various OTTR PR check or rendering steps.
```